samples/nrf9160/modem_shell/src/shell.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+
+#include <zephyr.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+#include <modem/lte_lc.h>
+
+#if defined(CONFIG_LWM2M_CARRIER)
+#include <lwm2m_carrier.h>
+#endif
+#include "at/at_shell.h"
+#if defined(CONFIG_MOSH_PING)
+#include "ping/icmp_ping_shell.h"
+#endif
+#if defined(CONFIG_MOSH_SOCK)
+#include "sock/sock_shell.h"
+#endif
+#if defined(CONFIG_POSIX_API)
+#include <sys/select.h>
+#else
+#if !defined(CONFIG_NET_SOCKETS_POSIX_NAMES)
+#include <posix/sys/select.h>
```

Comment:
Including just `<posix/sys/select.h>` and this is now inside `#if defined(CONFIG_MOSH_IPERF3)` because iperf_api.h is the only one needing it. I don't know if it should rather be added to iperf_api.h.

Reply:
@rlubos Please resolve this if OK.